applications/annotate_points.md

```diff
@@ -0,0 +1,314 @@
+```
+
+## Annotating with points
+We will annotate the features of interest using points in a napari Points layer. Each feature will be given a different label so that we can track them across frames. To achieve this, we will store the label in the `Points.properties` property in the 'label' key. We will instantiate the `Points` layer without any points. However, we will initialize `Points.properties` with the property values we will be using to annotate the images. To do so, we will define a properties dictionary with a key named `label` and values `labels`. The key, 'label', is the name of the property we are storing which feature of interest each point corresponds with. The values, 'labels', is the list of the names of the features we will annotating (defined above in the "point_annotator()" section). 
+
+```python
+properties = {'label': labels}
+```
+
+We then add the points layer to the viewer using the `viewer.add_points()` method. As discussed above, we will be storing which feature of interest each points corresponds to via the `label` property we defined in the `properties` dictionary. To visualize the feature each points represent, we set the edge color as a color cycle mapped to the `label` property (`edge_color='label'`). 
+
+```python
+properties = {'label': labels}
+points_layer = viewer.add_points(
+    properties=properties,
+    edge_color='label',
+    edge_color_cycle=COLOR_CYCLE,
+    symbol='o',
+    face_color='transparent',
+    edge_width=8,
+    size=12
+)
+```
+
+Note that we set the `edge_color_cycle` to `COLOR_CYCLE`. You can define your own color cycle as a list of colors. The colors can be defined as hex strings, standard color names or RGBA arrays. For example, the [category10 color pallete](https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#category10) would be:
+
+```python
+COLOR_CYCLE = [
+	'#1f77b4',
+	'#ff7f0e',
+	'#2ca02c',
+	'#d62728',
+	'#9467bd',
+	'#8c564b',
+	'#e377c2',
+	'#7f7f7f',
+	'#bcbd22',
+	'#17becf'
+]
+```
+
+Finally, we set the edge color to a color cycle:
+
+```python
+	points_layer.edge_color_mode = 'cycle'
+```
+
+## Adding a GUI for selecting points
+Next, we will use magicgui to add a dropdown menu for selecting which point we would like to add. [magicgui](https://github.com/napari/magicgui) is a library from the napari team for building GUIs from functions and works by applying function decorators. To make the a dropdown menu populated with the valid point labels, we simply define the function with the label as an input argument and then decorate it with the `magicgui()` decorator, passing the labels choice as `label={'choices': labels}` (recall that labels was passed to `point_annotator() as a list of the allowable labels).
+
+```python
+@magicgui(label={'choices': labels})
+def label_selection(label):
+    return label
+```
+
+To create the GUI, we call the label_selection. Gui() method.
+
+```python
+label_menu = label_selection.Gui()
```

Comment:
btw @tlambert03 why the choice `Gui` instead of `gui`?

Reply:
it's a class... so I went with caps.